add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
```

Comment:
- Observations: There is no error handling or logging mechanism beyond echoing error messages, which may not provide sufficient debug information.
- Recommendations: Implement a simple logging mechanism that stores errors to a file or output them conditionally only in a non-production environment.

Example:
```php
if (!mysqli_query($connection, $query)) {
    error_log("MySQL error: " . mysqli_error($connection));
    http_response_code(500);
    echo "An error occurred. Please try again later.";
}
```